pkg/kubelet/kubelet.go

```diff
@@ -435,6 +435,15 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 		myFlannelExperimentalOverlay = false
 	}
 
+	clusterDNSIps := strings.Split(kubeCfg.ClusterDNS, ",")
```

Comment:
IMO instead of having own logic of splitting the string, we can user StringSliceVar function @ https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L138